src/common/options/rgw.yaml.in

```diff
@@ -3783,6 +3783,32 @@ options:
     other form of policies that Amazon does, so if you are mirroring
     policies between RGW and AWS, you may wish to set this to false.
   default: true
+- name: rgw_perf_counters_cache
+  type: bool
+  level: dev
+  default: false
+  desc: If set to true, rgw creates labeled perf counters and stores them in an rgw specific labeled perf counters cache. The size of this cache and whether or not there is eviction can be configured as well.
+  see_also:
+  - rgw_perf_counters_cache_size
+  - rgw_perf_counters_cache_eviction
+  services:
+  - rgw
+- name: rgw_perf_counters_cache_eviction
+  type: bool
+  level: advanced
+  desc: Whether or not the labeled rgw perfcounters cache evicts the least recently updated counters once rgw_perf_counters_cache_size is reached
```

Comment:
is this option useful for your testing? i can't think of a reason that anyone would turn this off in production, when it could eat up all their memory

Reply:
Removed this config option and required eviction in the cache